test/uk/gov/hmrc/excisemovementcontrolsystemapi/fixture/MovementTestUtils.scala

```diff
@@ -28,6 +30,7 @@ trait MovementTestUtils {
                               consigneeId: Some[String]
                             ): GetMovementResponse = {
     GetMovementResponse(
+      MovementId(UUID.fromString("cfdb20c7-d0b0-4b8b-a071-737d68dede5e")),
```

Comment:
Should the response just have the value of the MovementId object returned as a string rather than the object which would mean a nested json structure again?

Reply:
We're still waiting on the other PR to be merged. Yeah good point let's wait for that to go in then we can fix this to match.